src/full/Agda/TypeChecking/Coverage.hs

```diff
@@ -930,7 +930,20 @@ fixTargetType sc@SClause{ scTel = sctel, scSubst = sigma } target = do
       [ "target type before substitution:" <+> pretty target
       , "             after substitution:" <+> pretty (applySplitPSubst sigma target)
       ]
-    return $ sc { scTarget = Just $ applySplitPSubst sigma target }
+
+    -- We update the target quantity to 0 for erased constructors.
+    f <- do
+      case tag of
+        SplitCon c -> do
+          q <- getQuantity <$> getConstInfo c
+          case q of
+            Quantity0{} -> return (q <>)
```

Comment:
Picky comment: Shouldn't this rather return `mapQuantity (q <>)` and then below use just `f` instead of `mapQuantity f`?